index.html

```diff
@@ -16,10 +16,14 @@
 <body>
     <div id='map'></div>
     <div class='map-overlay' id='features'>
+        <h3>Can my municipality supply itself with electricity from sun and wind?</h3>
         <h2>Renewable electricity autarky</h2>
```

Comment:
This used to be the old heading. Now that there is a new one, this should not be a heading anymore. Instead, it should appear with the hovering I suggest.

Reply:
good idea. if you like, change it. I cannot easily see where to change it. thanks